uniqueidentifier.go

```diff
@@ -0,0 +1,74 @@
+package mssql
+
+import (
+	"database/sql/driver"
+	"encoding/hex"
+	"errors"
+	"fmt"
+)
+
+type UniqueIdentifier [16]byte
```

Comment:
Can we call this UUID? Or is there a reason not to? I'm not sure I care too much, but I can never seem to type UniqueIdentifier right if I'm not paying attention...